src/installer/installer.go

```diff
@@ -327,3 +336,22 @@ func (i *installer) updateReadyMasters(nodes *v1.NodeList, readyMasters *[]strin
 	}
 	i.log.Infof("Found %d master nodes: %+v", len(nodes.Items), nodeNameAndCondition)
 }
+
+func (i *installer) cleanupInstallDevice() error {
+	vgName, err := i.ops.GetVGByPV(i.Device)
+	if err != nil {
+		return err
+	}
+
+	if vgName == "" {
```

Comment:
just to be sure if we don't have vgName there i sno need to run RemovePV?

Reply:
no, pv is usually created by default when we/somebody creates a VG. in addition PV is just a label on the disk and PV without any VG will not result in device busy